src/main/java/cs2103/v15_1j/jimjim/ui/TodayPaneController.java

```diff
@@ -73,17 +74,36 @@ public void refreshData(){
 		}
 
 		if(showOverdue){
-			rowFactory.showOverdue();
+			rowFactory.addLabel("Overdue", "header");
+			int noOfOverdue = rowFactory.showOverdue();
+
+			if(noOfOverdue == 0){
+				rowFactory.addLabel("Congratulations there are no Overdue Events or Tasks", "no-overdue-label");
+			}
+
+			rowFactory.addLabel("", "");
 		}
 
 		showFloatingTasks();
 	}
 
 	private void showFloatingTasks(){
+		rowFactory.addLabel("Floating Tasks", "header");
 		hasCompleted = rowFactory.showAllFloatingTasks(showCompleted);
 
 		int index = todayGridPane.getChildren().size();
 
+		if(displayList.getFloatingTasksList().size() == 0){
```

Comment:
Well personally I don't seeing a lot of "No x/y/z" messages like this. If there is such a message at all, it should be when there are absolutely nothing at all.